page/about-jquery.md

```diff
@@ -10,3 +10,5 @@
 }</script>
 
 Getting started with jQuery can be easy or challenging, depending on your experience with JavaScript, HTML, CSS, and programming concepts in general. In addition to these articles, you can read about the [history of jQuery](https://jquery.org/history/) and the [licensing terms](https://jquery.org/license/) that apply to jQuery projects. You can also [make a donation](https://jquery.org/donate/) to help the [jQuery team](https://jquery.org/team/) continue to improve jQuery.
+
+One important thing to know is that jQuery is just a __JavaScript library__. All the power of jQuery is accessed via JavaScript, so needless to say, it's an important language to learn. Having a basic knowledge of JavaScript will go a long way in understanding, structuring, and debugging your code. If you're new to JavaScript we recommend to check out the [JavaScript basics tutorial](https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/JavaScript_basics) on the Mozilla Developer Network (MDN).
```

Comment:
I think "important language to learn" kind of understates the importance. People constantly complain that jQuery is "the reason people don't *really* know JavaScript - this paragraph should be the thing where we state that we actually do feel like some command of JS is essentially a "prerequisite" for being effective with jQuery.